src/lib/libdwarf/CMakeLists.txt

```diff
@@ -81,9 +81,15 @@ if (BUILD_SHARED)
     endif()
     set(COMPILER_FLAGS -DLIBDWARF_BUILD -DPIC)
 else()
-    set(COMPILER_FLAGS -DLIBDWARF_STATIC)
+    set(COMPILER_FLAGS -DLIBDWARF_BUILD -DLIBDWARF_STATIC)
 endif()
 
+if (PIC_ALWAYS)
```

Comment:
If linking libdwarf statically against a PIC binary libdwarf will need to be PIC